HW04/Igor_Hrybinyk/spec/my_each_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../my_methods.rb'
+
+RSpec.describe Array do
+  subject { described_class.new([1, 2, 3, 4, 5]) }
+
+  describe '#my_each' do
+    let(:test_array) { [] }
+    let(:expected_result) { [2, 3, 4, 5, 6] }
+
+    context 'when works as default each' do
+      it 'returns valid values' do
+        subject.my_each { |elem| test_array << elem + 1 }
```

Comment:
We can use `before` block for this case 😉
```suggestion
    context 'when works as default each' do
      before { subject.my_each { |elem| test_array << elem + 1 } }
      it 'returns valid values' do
        
```